src/shared/ui/modal/modal.module.scss

```diff
@@ -0,0 +1,74 @@
+.modal {
+  max-width: 889px;
+  padding: 0 32px 32px 32px;
+
+  line-height: 24px;
+
+  border: none;
+  border-radius: 12px;
```

Comment:
We have squared corners on the right side when scroll is visible
![image](https://github.com/user-attachments/assets/d5dbdec8-ef22-4c26-8f75-a3e29ae2c452)